src/objects/tokens/mirage.js

```diff
@@ -0,0 +1,48 @@
+const { refObject, Rotator } = require("../../wrapper/api");
+const { System, Planet } = require("../../lib/system/system");
+
+const MIRAGE_ATTRS = {
+    localeName: "planet.mirage",
+    resources: 1,
+    influence: 2,
+    trait: ["cultural"],
+    position: { x: 2, y: -1.25 },
+    legendary: true,
+    legendaryCard: "card.legendary_planet:pok/mirage_flight_academy",
+};
+let MIRAGE_SYSTEM = 0;
```

Comment:
I'd suggest using `ALL_CAPS_FOR_CONSTANTS` and `_underscoreGlobalVar` for values that change.